Gruntfile.js

```diff
@@ -74,5 +88,12 @@ module.exports = function(grunt) {
   }
 
   grunt.registerTask("dev", ["connect", "watch"]);
-  grunt.registerTask("test", ["connect", "saucelabs-custom"]);
+  grunt.registerTask("test-locally", ["connect", "jasmine"]);
+
+  var secureEnvVars = process.env.TRAVIS_SECURE_ENV_VARS;
+  if (secureEnvVars === undefined || secureEnvVars === "true") {
```

Comment:
Yes, checking for `SAUCE_USERNAME` + `SAUCE_ACCESS_KEY` is more intuitive. Let's do this is possible, because it's no longer tied to Travis.